hs_core/forms.py

```diff
@@ -972,6 +972,7 @@ def __init__(self, allow_edit=True, res_short_id=None, element_id=None, element_
                             ),
                          )
         else:
+            self.form_tag = False
```

Comment:
This line may not be necessary since it has been set appropriately before in this function.